storagenode/peer.go

```diff
@@ -91,13 +89,11 @@ type Config struct {
 	Version version.Config
 
 	Bandwidth bandwidth.Config
-
-	Contact contact.Config
 }
 
 // Verify verifies whether configuration is consistent and acceptable.
 func (config *Config) Verify(log *zap.Logger) error {
-	return config.Kademlia.Verify(log)
+	return config.Operator.Verify(log)
```

Comment:
looks like the bootstrap peer has this same method. should that be updated to use Operator as well? Or who cares since we will remove bootstrap?

Reply:
I think we can leave it since I have a PR right after this to remove all the bootstrap stuff

Reply:
https://github.com/storj/storj/pull/2974

Reply:
ah-ha!